components/Card/index.js

```diff
@@ -28,12 +34,12 @@ const TYPE_URLS = {
   resident: "residents",
 };
 
-export default function Card({ name, type, slug, iconSource }) {
+export default function Card({ name, type, slug, iconSource, variant }) {
   const urlScheme = `${TYPE_URLS[type]}/${slug}`;
 
   return (
     <Link href={urlScheme} passHref legacyBehavior>
-      <StyledArticle>
+      <StyledArticle variant={variant && variant}>
```

Comment:
you don't need to check here if variant is defined or not. The && operator returns the first falsy value, in this case `undefined` if variant is not defined. Therefore it is the same as if you would just hand over variant directly.
```suggestion
      <StyledArticle variant={variant}>
```